tests/test_core.py

```diff
@@ -3351,14 +3351,14 @@ def test_dylink_funcpointers2(self):
           assert(((newSize+1) & 3) !== 1 || ((newSize+2) & 3) !== 2);
           loadDynamicLibrary('liblib.so');
         });
-        volatilevoidfunc f;
-        f = (volatilevoidfunc)left1;
+        voidfunc f;
+        f = (voidfunc)left1;
```

Comment:
what this volatile is trying to do is to avoid the optimizer turning an indirect call into a direct one, so that we actually test indirect calls here.

is that unreasonable?

Reply:
Ah yes.   I think that makes sense, I wonder if we can find a better way to ensure that.  At least I'll add a comment.

Reply:
Cool. I think most of these volatiles are there for similar reasons. So, not great reasons ;) but I worry about removing them.